assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -272,6 +290,10 @@ export default function StoryListView({
             </TableDateHeaderCell>
             <TableDateHeaderCell>
               <SelectableTitle
+                aria-label={__(
+                  'Date last modified, select to sort table by date story was last modified on',
```

Comment:
Grammar nit; ending this in a preposition might be confusing as the reader might expect more words

Reply:
sure - i'll remove 'on'. Was hoping there'd be input on what these should say, copywriting is not my forte.